apps/mull-ui/src/app/components/custom-file-upload/custom-file-upload.tsx

```diff
@@ -19,11 +20,17 @@ export const CustomFileUpload = ({
   errorMessage,
   handleFileUpload,
   fieldName,
+  isEditProfile,
 }: CustomFileUploadProps) => {
   return (
     <div className="custom-file-upload-container">
       <label htmlFor={fieldName} className={`${imageURL ? 'image-uploaded' : ''}`}>
-        {imageURL ? (
+        {imageURL && isEditProfile ? (
+          <>
+            <img className="custom-file-upload-image" src={imageURL} alt="Event" />
```

Comment:
The image is much larger than in the mockup. Personally I find it looks nice but check with the UI team. If they say this is okay make sure the mockup is updated

Reply:
Done. Reduced the image size.